Applications/VpView/vpVidtkTrackIO.cxx

```diff
@@ -284,7 +284,7 @@ bool vpVidtkTrackIO::WriteTracks(
   const auto& typesFilename = filestr + ".types";
   const auto& regionsFilename = filestr + ".regions";
 
-  std::string attributesFilename(filename);
+  std::string attributesFilename(filename.toStdString());
```

Comment:
This may use the wrong locale; please use `stdString(filename)` instead.

Reply:
done